modules/models.py

```diff
@@ -86,6 +87,24 @@ def load_model(model_name):
 
         return model, tokenizer
 
+    # 4-bit LLaMA
+    elif shared.args.load_in_4bit:
+        sys.path.append(os.path.abspath(Path("repositories/GPTQ-for-LLaMa")))
```

Comment:
Had to change this to insert instead of append when normal llama repo was installed as well, or it would load from that instead and not find load_quant